blog/electron-29-0.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Electron 29.0.0
+date: 2024-02-20T00:00:00.000Z
+authors:
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-29-0
+---
+
+Electron 29.0.0 has been released! It includes upgrades to Chromium `122.0.6261.39`, V8 `12.2`, and Node.js `20.9.0`.
+
+---
+
+The Electron team is excited to announce the release of Electron 29.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Stack Changes
+
+- Chromium `122.0.6261.39`
+  - New in [Chrome 122](https://developer.chrome.com/blog/new-in-chrome-122/) and in [DevTools 122](https://developer.chrome.com/blog/new-in-devtools-122/)
+  - New in [Chrome 121](https://developer.chrome.com/blog/new-in-chrome-121/) and in [DevTools 121](https://developer.chrome.com/blog/new-in-devtools-121/)
+- Node `20.9.0`
+  - [Node 20.9.0 notes](https://nodejs.org/en/blog/release/v20.9.0/)
+  - [Node 20.0.0 notes](https://nodejs.org/en/blog/release/v20.0.0/)
+- V8 `12.2`
+
+### New Features
+
+- Added net module to utility process. [#40890](https://github.com/electron/electron/pull/40890)
+- Added new [Electron Fuse](https://www.electronjs.org/docs/latest/tutorial/fuses) that opts the `file://` protocol into more secure and restrictive behaviour that matches Chromium. [#40372](https://github.com/electron/electron/pull/40372)
```

Comment:
We should probably name the fuse (`grantFileProtocolExtraPrivileges`) outright here.